src/test/java/pageObjects/listingDetails_page.java

```diff
@@ -0,0 +1,74 @@
+package pageObjects;
+
+import static org.testng.Assert.assertTrue;
+
+import org.openqa.selenium.By;
+
+import org.openqa.selenium.WebDriver;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import stepDefinitions.hooks;
+
+
+public class listingDetails_page {
+	
+	private WebDriver driver;
+	private static WebElement element = null;
+	
+	
+	public static void load_listings_details_page(WebDriver driver) {
+			driver = hooks.driver;
+			driver.get("https://chpl.ahrqstg.org/#/product/8481");
+			WebDriverWait wait = new WebDriverWait(driver, 20);
+			driver.manage().window().maximize();
+			String actualTitle = driver.getTitle();
+			System.out.println("page title is : "+actualTitle);
```

Comment:
These last two lines don't have anything to do with the stated reason for the method